### ISSUE TITLE
new issue screen not closing after submit

### ISSUE THREAD

**Issue by jouderianjr** [gitpoint/git-point#432]
This issue is related with new issue feature. After submit a new issue, the form screen is still in navigation stack, I believe new issue screen should be removed.

![newissueerrorreal](https://user-images.githubusercontent.com/1559013/31250414-1a4aea04-a9f1-11e7-9c23-c3264edfe5f3.gif)

**Comment by Antoine38660** [comment]
😱 this one was referenced here #236 and I fixed it in #243 !

I'll take a look ASAP dude :)

**Comment by Antoine38660** [comment]
How were you able to go back? The top left arrow isn't displayed (in my case) and the swipe left-to-back option is also disabled

<img width="559" alt="capture d ecran 2017-10-06 a 09 45 13" src="https://user-images.githubusercontent.com/9287184/31268094-1869df44-aa7b-11e7-8589-264ce9894adc.png">

**Comment by lex111** [comment]
@Antoine38660 On Android there is a button back, apparently through probably.

**Comment by jouderianjr** [comment]
@Antoine38660 yeah, Android has the back button, still, maybe remove the button is not the best approach because we are locking the whole stack, for example, the user is not able to back to issue list :/

**Comment by FlorianTaut** [comment]
Probably we should not remove the back button but set the link to the issue list insted to new issue Screen.
